common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
@cptvitamin "to review" sounds odd to me without a subject - I'd like to add something back to the end of the sentence. Maybe:
"Answers to this problem are now shown. Navigate through the problem to review it with answer text inline."?